src/HamamatsuCamera.cpp

```diff
@@ -2977,4 +2977,290 @@ void Camera::setHighDynamicRangeEnabled(const bool & in_enabled)
 HwEventCtrlObj* Camera::getEventCtrlObj()
 {
     return &m_event_ctrl_obj;
+}
+
+//=============================================================================
+// OUTPUT TRIGGER KIND
+//=============================================================================
+//-----------------------------------------------------------------------------
+/// Return the output trigger kind of given channel by the current detector
+//-----------------------------------------------------------------------------
+enum Camera::Output_Trigger_Kind Camera::getOutputTriggerKind(int channel)
+{
+    DEB_MEMBER_FUNCT();
+
+    DEB_TRACE() << " Camera::Output_Trigger_Kind Camera::getOutputTriggerKind(int channel) : ..."; 
+    enum Output_Trigger_Kind result = Output_Trigger_Kind::Output_Trigger_Kind_Not_Supported;
+
+    DCAMERR err;
+    double kindArraySize = 0;
+    enum Output_Trigger_Kind kind = Camera::Output_Trigger_Kind_Not_Supported;
+
+	// get property attribute that contains the Kind (that may be an array)
+	DCAMPROP_ATTR	basepropattr;
+	memset(&basepropattr, 0, sizeof(basepropattr));
+	basepropattr.cbSize = sizeof(basepropattr);
+	basepropattr.iProp = DCAM_IDPROP_OUTPUTTRIGGER_KIND;
+	err = dcamprop_getattr(m_camera_handle, &basepropattr);

```

Comment:
1/ could this code be replaced by the getPropertyData function ?
2/ fix the formatting: ie start is not aligned